### PR TITLE
Update troubleshooting.mdx

### PR DESCRIPTION
**Pull request by RMartinOscar**
+ Added 502 to common panel errors
+ Added invalid CSRF to common panel errors
+ Added invalid MAC to common panel errors

+ Added pool overlaps to common wings errors

- Removed useless /* at the end of chown commands as they already use the -R arg

used the groupId feature of Docusaurus so users that already selected webserver in getting-started are directly on the right tab)

### CODE REVIEW

File: docs/troubleshooting.mdx [hunk 1]
```diff
@@ -9,29 +9,32 @@ import TabItem from '@theme/TabItem';
 # Troubleshooting
 ## Panel errors
 
-If you see a `500 | SERVER ERROR` or `An unexpected error was encountered while processing this request` you have to check your panel logs by running the following command.
-
+* `502 Bad Gateway Error` or some code in plain text you either didn't install php-fpm or set the wrong version in your webserver vhost.
+* `500 | SERVER ERROR` or `An unexpected error was encountered while processing this request` you have to check your panel logs by running the following command.
+* `CSRF token mismatch` you are most likely using HTTP instead of HTTPS in `APP_URL` or you didn't force `SESSION_SECURE_COOKIE` to false while using HTTP.
```
Comment (Boy132) [inline docs/troubleshooting.mdx:14]: Please move the 502 error and CSRF error to the "common errors" section. The introductory text should stay.

File: docs/troubleshooting.mdx [hunk 2]
```diff
@@ -9,29 +9,32 @@ import TabItem from '@theme/TabItem';
...
 ```sh
 tail -n 100 /var/www/pelican/storage/logs/laravel-$(date +%F).log | grep "\[$(date +%Y)"
 ```
 
+If it returns `The MAC is invalid` make sure that your `APP_KEY` matchs your database, if its a clean install just clear it then re run `php artisan p:environment:setup`
```
Comment (Boy132) [inline docs/troubleshooting.mdx:19]: Same here. Move to "common errors".

File: docs/troubleshooting.mdx [hunk 3]
```diff
@@ -55,6 +58,7 @@ It should say that the service is active and running. If its in a failed state r
 Some common errors in the wings logs are:
 
 * `open /etc/letsencrypt/live/_____________/fullchain.pem: no such file or directory`: The SSL certificate for wings is missing, see [this guide](./guides/ssl) for creating a SSL certificate.
+* `response from daemon: Pool overlaps with other one on this address space` : You already have a docker network using the 172.18.0.0/16 subnet please edit your config file or remove it.
```
Comment (Boy132) [inline docs/troubleshooting.mdx:61]:
```suggestion
* `Pool overlaps with other one on this address space` : You already have a docker network using the default subnet. (`172.18.0.0/16`) Change the subnet in your wings config file and use `systemctl stop wings && docker network rm pelican_nw && systemctl start wings` to apply the change.
```

**Comment by RMartinOscar** [comment]
Hey i moved the errors to their corresponding categories

**Pull request merged**